python/lsst/sims/photUtils/SignalToNoise.py

```diff
@@ -307,62 +306,40 @@ def calcSNR_m5(magnitudes, bandpasses, m5, photParams, gamma=None):
     @param [in] gamma (optional) is the gamma parameter from equation(5) of
     arXiv:0805.2366.  If not provided, this method will calculate it.
 
-    @param [out] snr is a numpy array of the signal to noise ratio corresponding to
-    the input magnitudes.
-
-    @param [out] gamma is a numpy array of the calculated gamma parameters for the
-    bandpasses used here (in case the user wants to call this method again.
-    """
-
-    if magnitudes.shape[0] != len(bandpasses):
-        raise RuntimeError("Passed %d magnitudes to " % magnitudes.shape[0] + \
-                            " calcSNR_m5; " + \
-                            "passed %d bandpasses" % len(bandpasses))
+    @param [out] snr is the signal to noise ratio corresponding to
+    the input magnitude.
 
-    if gamma is not None and len(gamma) != len(bandpasses):
-        raise RuntimeError("Passed %d bandpasses to " % len(bandpasses) + \
-                           " calcSNR_m5; " + \
-                           "passed %d gamma parameters" % len(gamma))
+    @param [out] gamma is  the calculated gamma parameter for the
+    bandpass used here (in case the user wants to call this method again).
 
-    if len(m5) != len(bandpasses):
-        raise RuntimeError("Passed %d bandpasses to " % len(bandpasses) + \
-                           " calcSNR_m5; " + \
-                           "passed %d m5 values" % len(m5))
+    Note: You can also pass in a numpy array of magnitudes calculated
+    in the same bandpass and get a numpy array of SNR out
```

Comment:
Does this mean that all the magnitudes are for the same m5 and gamma (ie. single pointing many sources)